roles/openshift_cli/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-openshift_version: "{{ openshift_image_tag | default(openshift.common.image_tag) | default('') }}"
+openshift_version: "{{ openshift_image_tag | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
I actually really like this moving under the 'docker' role, do we need to migrate the existing common.image_tag value to docker.openshift_image_tag in openshift_facts though?